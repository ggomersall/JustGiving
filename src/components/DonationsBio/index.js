import React from 'react';

const DonationsBio = ({charity}) => {

  return(
    <div class>
      {charity.pageShortName}
    </div>
  )
}

export default DonationsBio