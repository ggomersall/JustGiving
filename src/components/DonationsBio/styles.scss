@import '../BaseStyles/variables';

.donation-bio__wrapper {
  width: 50%;
  border-bottom: 2px solid $grey;
  margin-bottom: 15px;
} 