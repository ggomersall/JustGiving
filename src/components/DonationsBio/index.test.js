import React from 'react';
import { shallow } from 'enzyme';
import DonationsBio from './index.js';

describe('Testing DonationsBio Component', () => {

  let wrapper;

  test('it should render data correctly', () => {
    const charity = {
      name: 'Macmillan',
      description: 'lorem ipsum text',
      websiteUrl: 'http://www.justgiving.com',
      emailAddress: 'macmillan@justgiving.com',
      registeredNumber: 12344
    };
    wrapper = shallow(<DonationsBio charity={charity} />);
  });

  test('Snapshot', () => expect(<DonationsBio />).toMatchSnapshot());

});

