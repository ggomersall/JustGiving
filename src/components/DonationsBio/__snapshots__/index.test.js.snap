// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DonationsBio Component Snapshot 1`] = `<DonationsBio />`;
